modules.d/73zipl/module-setup.sh

```diff
@@ -61,6 +61,6 @@ install() {
         local _zipl
         _zipl=$(cmdline)
 
-        [[ $_zipl ]] && printf "%s\n" "$_zipl" > "${initdir}/etc/cmdline.d/91zipl.conf"
+        [[ $_zipl ]] && printf "%s\n" "$_zipl" > "${initdir}/etc/cmdline.d/20-zipl.conf"
```

Comment:
It looks fishy that you're using the same file name her as in install_zipl_cmdline.sh above, while previously the file names were not identical. At least a comment would be in order if you have verified that this is correct.